libafl_qemu/librasan/asan/src/maps/iterator.rs

```diff
@@ -0,0 +1,145 @@
+use alloc::string::String;
+use core::fmt::Debug;
+
+use crate::{
+    GuestAddr,
+    maps::{MapReader, decode::MapDecode, entry::MapEntry},
+};
+
+#[derive(Debug)]
+enum MapState {
+    Base,
+    Limit,
+    Properties,
+    Offset,
+    Major,
+    Minor,
+    Inode,
+    Path,
+}
+
+const BUFFER_SIZE: usize = 4096;
```

Comment:
Put the consts/variable above enum/struct/.. declarations

Reply:
Done